### PR TITLE
Added support for bigint in ClassNameValue type

### PR DESCRIPTION
**Pull request by LarsArtmann**
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-2.html#bigint

### CODE REVIEW

File: src/lib/tw-join.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * Original code has MIT license: Copyright (c) Luke Edwards <luke.edwards05@gmail.com> (lukeed.com)
  */
 
-export type ClassNameValue = ClassNameArray | string | null | undefined | 0 | false
+export type ClassNameValue = ClassNameArray | string | null | undefined | 0 | false | bigint
```
Comment (dcastil) [inline src/lib/tw-join.ts:11]: Only falsy bigints should be allowed here. Could you change it to only allow `0n` instead?

**Review by dcastil — CHANGES_REQUESTED** [review]
Thanks for your PR! Just needs a small change.

**Comment by LarsArtmann** [comment]
Should be fine now.

**Review by dcastil — APPROVED** [review]
:rocket:

**Pull request merged**